### DISCUSSION TITLE
Malcolm Howard, Fortunato Hernandez

### DISCUSSION THREAD

**Discussion by fortuhf** [birkenkrahe/ai482#15] [Show and tell]
https://drive.google.com/file/d/1oEDH9kpcaMIllYcp-7zoOh2toPGcDPfA/view?usp=sharing

**Comment by birkenkrahe** [comment]
- Of course I've seen a lot of sports analytics this last term - I
   feel better prepared for your presentation now! I liked the
   breakdown of measures and the detailed explanation.
   - The "Route Recognition" measure in particularly made me want to see
   a vizualization across the league. I wonder if it would be possible
   to predict routes from these data.
- The new thing was the comparative analysis - the games are quite
   different, of course, and this shows in the tools. I wonder if the
   similarities or the differences are more noteworthy. "AI" as a
   common denominator is a little rough.
- Really liked the bias/flaws analysis. Good, creative ideas and
   critical thinking! If something's a "flaw" depends of course what
   you're looking for.
- References are not complete or consistent!
- A little reflection on the project wouldn't have hurt.
